### PR TITLE
[markdown] Fix lint errors in `packages/i18n-calypso-cli`

### PR DESCRIPTION
**Pull request by scinos**
### Background

We want to lint the code blocks inside Markdown files to follow our coding style.

### Changes

This PR fixes all markdown errors in `packages/i18n-calypso-cli`

### Testing instructions

Run `./node_modules/.bin/eslint --ext .md --ext .md.js --ext .md.javascript --ext .md.jsx packages/i18n-calypso-cli`, there should be no errors

### CODE REVIEW

File: packages/i18n-calypso-cli/README.md [hunk 1]
```diff
@@ -14,12 +14,12 @@ Scans your JavaScript sources/build files and generates a POT file or a PHP tran
...
 
 ```js
-var i18nCalypso = require( 'i18n-calypso-cli' );
+const i18nCalypso = require( 'i18n-calypso-cli' );
 i18nCalypso( {
-    inputPaths: <paths to your js files to scan>,
-    output: <path to your destination>
-    format: <format of the output: POT, PHP or JSON>
-    projectName: <Meta information about the project which can be used for autogenerated headers>
+	inputPaths, //<paths to your js files to scan>,
```
Comment (tyxla) [inline packages/i18n-calypso-cli/README.md:19]: I'd generally prefer a space between the `//` and the actual comment. I'd expect linting to catch that IMHO. 

Not a huge blocker though.

File: packages/i18n-calypso-cli/.eslintrc.js [hunk 2]
```diff
@@ -6,4 +6,12 @@ module.exports = {
 		'import/no-nodejs-modules': 0,
 		'import/no-extraneous-dependencies': [ 'error', { packageDir: __dirname } ],
 	},
+	overrides: [
+		{
+			files: [ '*.md.js' ],
```
Comment (tyxla) [inline packages/i18n-calypso-cli/.eslintrc.js:11]: `*.md.js` was intentional, right? We're targeting JS blocks inside markdown files?

**Review by tyxla — APPROVED** [review]
LGTM, I've only added a couple of questions.

File: packages/i18n-calypso-cli/README.md (hunk 1 above)
Comment (scinos) [reply packages/i18n-calypso-cli/README.md:19]: Eslint to the rescue! We could enable https://eslint.org/docs/rules/spaced-comment

File: packages/i18n-calypso-cli/.eslintrc.js (hunk 2 above)
Comment (scinos) [reply packages/i18n-calypso-cli/.eslintrc.js:11]: Yes. The plugin we are using uses this format to specify "codeblocks of language 'js' inside markdown" (https://www.npmjs.com/package/eslint-plugin-md#modifying-eslint-setup-for-js-code-inside-md-files)

The architecture is actually quite clever: the plugin uses an eslint preprocessor to receive a single Markdown file as input and produces one Markdown without code blocks + n files named 'file.md.<lang>' for each code block. Then lints each one of those individually, using the regular JavaScript rules to lint `file.md.js`. And then aggregates all the errors from each "sub-file" tweaking line and column numbers so it makes sense as a whole.

File: packages/i18n-calypso-cli/README.md (hunk 1 above)
Comment (tyxla) [reply packages/i18n-calypso-cli/README.md]: I'm in favor, but it's work for another PR I'd expect. No need to flood this one.

**Pull request merged**